eva/udfs/emotion_detector.py

```diff
@@ -0,0 +1,213 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+from typing import List
+
+import numpy as np
+import pandas as pd
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchvision.transforms as transforms
+from PIL import Image
+from skimage.transform import resize
+from torch.autograd import Variable
+
+from eva.configuration.dictionary import EVA_DEFAULT_DIR
+from eva.udfs.abstract_udf import AbstractClassifierUDF
+from eva.udfs.gpu_compatible import GPUCompatible
+
+# VGG configuration
```

Comment:
What kind of configuration is this? Is this the structure of the neural network? A small comment would be nice

Reply:
Yes, it specifies the architecture. But yes, let me clean it up a little.